translate/misc/test_xml_helpers.py

```diff
@@ -0,0 +1,36 @@
+from lxml import etree
+
+from translate.misc.xml_helpers import reindent
+
+class TestCleanIndent:
+    def _xmlfromstring(self, xmlstring):
+        return etree.fromstring(xmlstring)
+
+    def _xmltostring(self, xml):
+        return etree.tostring(xml, pretty_print=True, xml_declaration=True, encoding='utf-8')
+
+    def test_indent_four_spaces(self):
+        """Test that using 4 spaces for indent yields a consistent result."""
+        xmlsource = self._xmlfromstring('<root><str key="test">Test</str></root>')
+        reindent(xmlsource, indent="    ")
+        actual = self._xmltostring(xmlsource)
+        expected = (
+            "<?xml version='1.0' encoding='utf-8'?>\n"
+            '<root>\n'
+            '    <str key="test">Test</str>\n'
+            '</root>\n'
```

Comment:
Here you have enclosing double quotes in first line of implicit string continuation, but single quotes in the following three lines.

Reply:
Same below.

Reply:
I now see that below we are testing tab indenting, and using triple quoted string might hide the presence of a tab in the expected string.

Anyway I find weird to have different quoting along the string. As long as it works ok, I am fine to let this one go.